src/accelerate/inference.py

```diff
@@ -29,8 +29,7 @@
 
 
 if is_pippy_available():
-    from pippy.IR import Pipe, PipeSplitWrapper, annotate_split_points
-    from pippy.PipelineStage import PipelineStage
+    from pippy import Pipe, PipelineStage, SplitPoint, annotate_split_points
```

Comment:
Let's modify `is_pippy_available` to give a `min_version` flag, then do an if/else to bring in the right imports